SRC/classes/StudentQuery.cls

```diff
@@ -0,0 +1,10 @@
+public class StudentQuery {
+    public StudentQuery(){
+        List<Student__c> StudentList = [select id, Name,Roll_Number__c,Father_Name__c,Father_Mobile_Number__c,
+                                        HomePhone__c,Email__c,Address__c,Overall_Grade__c
+                                        from Student__c order by createdDate Limit 5];
+        	for(Student__c Teacher: StudentList){
+            	System.debug('Check data ::: '+Teacher);
```

Comment:
While Adding the code changes for PR please review that no debug statements are present.